# Solaris-specifc core file processing

--- gdb-8.0-orig/bfd/elf-bfd.h	2017-06-04 08:54:54.000000000 -0700
+++ gdb-8.0/bfd/elf-bfd.h	2017-06-14 10:17:25.000000000 -0700
@@ -1253,6 +1253,11 @@ struct elf_backend_data
   bfd_boolean (*elf_backend_grok_psinfo)
     (bfd *, Elf_Internal_Note *);
 
+  /* This function, if defined, is called when an NT_LWPSTATUS
+     note is found in a core file.  */
+  bool (*elf_backend_grok_lwpstatus)
+    (bfd *, Elf_Internal_Note *);
+
   /* This function, if defined, is called to write a note to a corefile.  */
   char *(*elf_backend_write_core_note)
     (bfd *abfd, char *buf, int *bufsiz, int note_type, ...);

