
--- gdb-8.0-orig/bfd/elfxx-target.h	2017-06-04 08:54:54.000000000 -0700
+++ gdb-8.0/bfd/elfxx-target.h	2017-07-10 15:57:40.000000000 -0700
@@ -585,6 +585,9 @@
 #ifndef elf_backend_grok_psinfo
 #define elf_backend_grok_psinfo			NULL
 #endif
+#ifndef elf_backend_grok_lwpstatus
+#define elf_backend_grok_lwpstatus              NULL
+#endif
 #ifndef elf_backend_write_core_note
 #define elf_backend_write_core_note		NULL
 #endif
@@ -808,6 +811,7 @@ static struct elf_backend_data elfNN_bed F
   elf_backend_sort_relocs_p,
   elf_backend_grok_prstatus,
   elf_backend_grok_psinfo,
+  elf_backend_grok_lwpstatus,
   elf_backend_grok_freebsd_prstatus,
   elf_backend_write_core_note,
   elf_backend_lookup_section_flags_hook,

